sdk/selfhelp/Azure.ResourceManager.SelfHelp/src/Generated/Models/StepInput.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.ResourceManager.SelfHelp.Models
+{
+    /// <summary> Details of step input. </summary>
+    public partial class StepInput
+    {
+        /// <summary> Initializes a new instance of StepInput. </summary>
+        internal StepInput()
+        {
+            ResponseOptions = new ChangeTrackingList<ResponseOption>();
+        }
+
+        /// <summary> Initializes a new instance of StepInput. </summary>
+        /// <param name="questionId"> Use Index as QuestionId. </param>
+        /// <param name="questionType"> Text Input. Will be a single line input. </param>
+        /// <param name="questionContent"> User question content. </param>
+        /// <param name="questionContentType"> Default is Text. </param>
+        /// <param name="responseHint"> Place holder text for response hints. </param>
+        /// <param name="recommendedOption"> Result of Automate step. </param>
+        /// <param name="selectedOptionValue"> Text of response that was selected. </param>
+        /// <param name="responseValidationProperties"> Troubleshooter step input response validation properties. </param>
+        /// <param name="responseOptions"></param>
+        internal StepInput(string questionId, string questionType, string questionContent, QuestionContentType? questionContentType, string responseHint, string recommendedOption, string selectedOptionValue, ResponseValidationProperties responseValidationProperties, IReadOnlyList<ResponseOption> responseOptions)
+        {
+            QuestionId = questionId;
+            QuestionType = questionType;
+            QuestionContent = questionContent;
+            QuestionContentType = questionContentType;
+            ResponseHint = responseHint;
+            RecommendedOption = recommendedOption;
+            SelectedOptionValue = selectedOptionValue;
+            ResponseValidationProperties = responseValidationProperties;
+            ResponseOptions = responseOptions;
+        }
+
+        /// <summary> Use Index as QuestionId. </summary>
+        public string QuestionId { get; }
```

Comment:
What's the format of this property value?

Reply:
this is string